src/main/java/com/bootcamp/demo/model/Account.java

```diff
@@ -8,6 +13,8 @@
     private String phoneNumber;
     private String address;
     private String dateOfBirth;
+    @JsonIgnore
+    private ArrayList<Promotion> promotions = new ArrayList<>();
```

Comment:
Just an idea. Maybe store here only the promoCodes for simplity? Also, should this be a Set rather than a List?